src/api/emails/send-application-submitted-emails/index.ts

```diff
@@ -0,0 +1,124 @@
+import { Context } from '.keystone/types'; // eslint-disable-line
+import sendEmail from '../index';
+import getExporterById from '../../helpers/get-exporter-by-id';
+import { ANSWERS, EMAIL_TEMPLATE_IDS } from '../../constants';
+import { SubmitApplicationResponse, ApplicationSubmissionEmailVariables } from '../../types';
+
+/**
+ * applicationSubmittedEmails.send
+ * Send "application submitted" emails
+ * @param {Object} GraphQL root variables
+ * @param {Object} GraphQL variables for the SendEmailConfirmEmailAddress mutation
+ * @param {Object} KeystoneJS context API
+ * @returns {Object} Object with success flag and emailRecipient
+ */
+const send = async (
+  context: Context,
+  referenceNumber: number,
+  accountId: string,
+  buyerId: string,
+  declarationId: string,
+): Promise<SubmitApplicationResponse> => {
+  try {
+    // get the application's exporter
+    const exporter = await getExporterById(context, accountId);
+
+    // ensure that we have found an acount with the requsted ID.
+    if (!exporter) {
+      console.error('Sending application submitted emails to exporter - no exporter exists with the provided ID');
+
+      return {
+        success: false,
+      };
+    }
+
+    // get the application's buyer
+    const buyer = await context.db.Buyer.findOne({
+      where: { id: buyerId },
+    });
+
+    if (!buyer) {
+      console.error('Sending application submitted emails to exporter - no buyer exists with the provided ID');
+
+      return {
+        success: false,
+      };
+    }
+
+    // get the application's declarations
+    const declaration = await context.db.Declaration.findOne({
+      where: { id: declarationId },
+    });
+
+    if (!declaration) {
+      console.error('Sending application submitted emails to exporter - no declarations exist with the provided ID');
+
+      return {
+        success: false,
+      };
+    }
+
+    // generate email variables
+    const { email, firstName } = exporter;
```

Comment:
I guess `email` and `firstName` will always exist?

Reply:
Yes, they will always exist - an account/exporter cannot be created without these, amongst other fields